package.json

```diff
@@ -40,12 +40,9 @@
     "commander": "~8.3.0",
     "deep-extend": "~0.6.0",
     "get-stdin": "~8.0.0",
-    "glob": "~7.2.0",
```

Comment:
I think this is a typo

Reply:
Typo? I don't understand.

Reply:
glob is still used.

Reply:
It's a direct dependency. Try running the code after "npm install --production".

Reply:
Oh right, for some reason I was convinced newer versions of NodeJS come with a 1st party glob module like Python.